apps/opik-frontend/src/components/pages/DatasetItemsPage/DatasetItemsActionsPanel.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useRef, useState } from "react";
+import { Trash } from "lucide-react";
+
+import { Button } from "@/components/ui/button";
+import { DatasetItem } from "@/types/datasets";
+import useDatasetItemBatchDeleteMutation from "@/api/datasets/useDatasetItemBatchDeleteMutation";
+import ConfirmDialog from "@/components/shared/ConfirmDialog/ConfirmDialog";
+import TooltipWrapper from "@/components/shared/TooltipWrapper/TooltipWrapper";
+
+type DatasetItemsActionsPanelsProps = {
+  datasetItems: DatasetItem[];
+};
+
+const DatasetItemsActionsPanel: React.FunctionComponent<
+  DatasetItemsActionsPanelsProps
+> = ({ datasetItems }) => {
+  const resetKeyRef = useRef(0);
+  const [open, setOpen] = useState<boolean>(false);
+  const disabled = !datasetItems?.length;
+
+  const { mutate } = useDatasetItemBatchDeleteMutation();
+
+  const deleteDatasetItemsHandler = useCallback(() => {
+    mutate({
+      ids: datasetItems.map((i) => i.id),
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [datasetItems]);
```

Comment:
i would add `mutate` to the dependencies

Reply:
Fixed